scripts/plot_summary.py

```diff
@@ -385,9 +385,10 @@ def plot_carbon_budget_distribution():
     ax1.set_xlim([1990,snakemake.config['scenario']['planning_horizons'][-1]+1])
 
     path_cb = snakemake.config['results_dir'] + snakemake.config['run'] + '/csvs/'
-    countries=pd.read_csv(path_cb + 'countries.csv',  index_col=1)
+    pop_layout = pd.read_csv(snakemake.input.clustered_pop_layout, index_col=0)
```

Comment:
pop_layout is not used, is it?

Reply:
oh yes, forgot about removing the pop_layout again